contracts/src/v0.8/keystone/KeystoneForwarder.sol

```diff
@@ -83,25 +83,30 @@ contract KeystoneForwarder is IForwarder, ConfirmedOwner, TypeAndVersionInterfac
 
   /// @notice Emitted when a report is processed
   /// @param receiver The address of the receiver contract
-  /// @param workflowOwner The address of the workflow owner
   /// @param workflowExecutionId The ID of the workflow execution
+  /// @param reportName The name of the report
+  /// @param workflowOwner The address of the workflow owner
+  /// @param workflowName The name of the workflow
   /// @param result The result of the attempted delivery. True if successful.
   event ReportProcessed(
     address indexed receiver,
-    address indexed workflowOwner,
     bytes32 indexed workflowExecutionId,
+    bytes32 indexed reportName,
```

Comment:
I'm worried about how much extra metadata we're passing through now and any potential size issues on Solana. Could report name just be a `u8` type ID/discriminator? `255` different report types per workflow ought to be enough

Reply:
As discussed offline, we could use address (20 bytes) for owner and only 12 bytes for report name.